src/transactions/feed/detailContent/EarnContent.tsx

```diff
@@ -18,18 +17,19 @@ interface EarnClaimRewardProps {
 
 export function EarnClaimContent({ transaction }: EarnClaimRewardProps) {
   const { t } = useTranslation()
-  const { providerName } = getDynamicConfigParams(
-    DynamicConfigs[StatsigDynamicConfigs.EARN_STABLECOIN_CONFIG]
-  )
+  const pools = useSelector(earnPositionsSelector)
+  const providerName = pools.find((pool) => pool.appId === transaction.providerId)?.appName
```

Comment:
nit: maybe add a reusable hook for this, similar to https://github.com/valora-inc/wallet/blob/9a2df7d045d39f8a0b24adc6a1721f18b9c9daa6/src/earn/hooks.ts#L97-L104

Reply:
that makes sense! added in [cd2a03f](https://github.com/valora-inc/wallet/pull/5829/commits/cd2a03fbd2828613ce7cac1ebdc0e283501a38e4)